core/src/test/java/io/parsingdata/metal/ReferenceValueExpressionSemanticsTest.java

```diff
@@ -84,4 +90,13 @@ private static Token refList(final String first, final String second, final Valu
                    def("z", con(1), eq(exp)));
     }
 
+    private static final Token refAny = any("a");
+    private final static Reducer addReducer = new Reducer() { @Override public ValueExpression reduce(final ValueExpression l, final ValueExpression r) { return add(l, r); } };
```

Comment:
I've also added this one again in my PR, maybe make it available as a public constant somewhere?

Also, it should be `ADD_REDUCER`, since it's a constant.